packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -168,12 +176,20 @@ export class AbuseService {
           abuseEntity.chainId,
           abuseEntity.escrowAddress,
         );
-        const webhookUrl = (
-          await OperatorUtils.getOperator(
-            abuseEntity.chainId,
-            escrow.exchangeOracle as string,
-          )
-        ).webhookUrl as string;
+        if (!escrow) {
+          throw new Error(
+            `Escrow data is missing. Abuse id: ${abuseEntity.id}`,
+          );
+        }
+        const operator = await OperatorUtils.getOperator(
+          abuseEntity.chainId,
+          escrow.exchangeOracle as string,
+        );
+        if (!operator || !operator.webhookUrl) {
+          throw new Error(
+            `Operator data is missing. Abuse id: ${abuseEntity.id}`,
+          );
+        }
```

Comment:
```suggestion
        if (!operator) {
          throw new Error('Operator data is missing');
        }
```
And if needed - add another check and error for `operator.webhookUrl` so we can distinguish if it's missing data in subgraph (e.g. it lags) or just webhook url